maat-court-data-api/src/main/java/gov/uk/courtdata/reporder/service/RepOrderService.java

```diff
@@ -101,8 +101,11 @@ public void update(Integer repId, Map<String, Object> repOrder) {
     }
 
     @Transactional
-    public boolean exists(Integer repId) {
+    public boolean exists(Integer repId, boolean hasSentenceOrderDate) {
         log.info("Retrieve rep Order Count With Sentence Order Date - Transaction Processing - Start");
```

Comment:
Really minor thing, but wonder if it would be useful to print the `hasSentenceOrderDate` in the logs just to be clearer which query has been run?

Reply:
Thanks. Done now